src/components/ProgressCircle.ts

```diff
@@ -116,6 +121,15 @@ export class ProgressCircle extends Component<ProgressCircleProps, { alertMessag
                 progressText = `${value}`;
             } else if (text === "percentage") {
                 progressText = progress + "%";
+            } else if (text === "static" && this.props.displayTextStatic) {
+                progressText = !(this.props.displayTextStatic.includes("{value}"))
+                    ? `${progress} ${this.props.displayTextStatic}`
+                    : this.props.displayTextStatic
+                        .replace("{value}", String(value))
+                        .replace("{maximumValue}", String(this.props.maximumValue));
+
+            } else if (text === "attribute" && displayTextValue) {
```

Comment:
Also said this could contain a template... don't see why not.

Reply:
Changed